### ISSUE TITLE
Adjustable font/text size for the settings menu

### ISSUE THREAD

**Issue by Smannesman** [luciusDXL/TheForceEngine#426]
As someone with less than stellar eyesight I appreciate the accessibility options built-in to TFE a lot, but the settings screen itself uses a font size I'm unable to easily read.
It's a very minor issue since it's not a screen that one often needs to access, but if you should ever have a bit of additional time to maybe make it a configurable option or just increase the size by 2-3 points I'd certainly appreciate it.

**Comment by luciusDXL** [comment]
The settings UI already supports different font sizes based on resolution (so it is larger at 1440p, 4k, etc.) - so I need to expose this as a general user option (and test it still works correctly).
